mon-pix/tests/integration/components/scorecard-details-test.js

```diff
@@ -146,158 +97,209 @@ describe('Integration | Component | scorecard-details', function() {
+    context('When the user did not started a competence', function() {
+      it('should not display the level and remainingPixToNextLevel if scorecard.isNotStarted is true', async function() {
+        // given
+        const scorecard = {
+          remainingPixToNextLevel: 1,
+          isNotStarted: true,
+        };
 
-      // when
-      await render(hbs`{{scorecard-details scorecard=scorecard}}`);
+        this.set('scorecard', scorecard);
 
-      // then
-      const element = this.element.querySelector('.scorecard-details__resume-or-start-button');
-      expect(element).to.exist;
-      expect(element.textContent).to.contain('Reprendre');
-    });
+        // when
+        await render(hbs`{{scorecard-details scorecard=scorecard}}`);
 
-    it('should not display the tutorial section when there is no tutorial to show', async function() {
-      // given
-      const scorecard = {
-        competenceId: 1,
-        isStarted: true,
-      };
+        // then
+        expect(this.element.querySelector('.scorecard-details-content-right__score-container')).to.not.exist;
+        expect(this.element.querySelector('.scorecard-details-content-right__level-info')).to.not.exist;
+      });
 
-      this.set('scorecard', scorecard);
+      it('should display a button stating "Commencer" if scorecard.isStarted is false', async function() {
+        // given
+        const scorecard = {
+          competenceId: 1,
+          isStarted: false,
+        };
 
-      // when
-      await render(hbs`{{scorecard-details scorecard=scorecard}}`);
+        this.set('scorecard', scorecard);
 
-      // then
-      expect(find('.tutorials')).to.not.exist;
-    });
+        // when
+        await render(hbs`{{scorecard-details scorecard=scorecard}}`);
 
-    it('should display the tutorial section and the related tutorials', async function() {
-      // given
-      const tuto1 = EmberObject.create({
-        title: 'Tuto 1.1',
-        tubeName: '@first_tube',
-        tubePracticalTitle: 'Practical Title',
-        duration: '00:15:10',
-      });
-      const tuto2 = EmberObject.create({
-        title: 'Tuto 2.1',
-        tubeName: '@second_tube',
-        tubePracticalTitle: 'Practical Title',
-        duration: '00:04:00',
+        // then
+        const element = this.element.querySelector('.scorecard-details__resume-or-start-button');
+        expect(element).to.exist;
+        expect(element.textContent).to.contain('Commencer');
       });
-      const tuto3 = EmberObject.create({
-        title: 'Tuto 2.2',
-        tubeName: '@second_tube',
-        tubePracticalTitle: 'Practical Title',
-        duration: '00:04:00',
-      });
-
-      const tutorials = A([tuto1, tuto2, tuto3]);
+    });
 
-      const scorecard = EmberObject.create({
-        competenceId: 1,
-        isStarted: true,
-        tutorials,
-      });
+    context('When the user has started a competence, i.e. scorecard.isStarted is true', async function() {
```

Comment:
la précision `scorecard.isStarted is true` dans le message me semble inutile, surtout qu'elle couple le message du test à un détail d'implémentation très fort

Reply:
Done ✅